swirl/build.gradle

```diff
@@ -1,17 +1,18 @@
 buildscript {
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.1.0'
+    classpath 'com.android.tools.build:gradle:3.0.1'
   }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-  compileSdkVersion 23
-  buildToolsVersion '23.0.3'
+  compileSdkVersion 27
+  buildToolsVersion '27.0.3'
 
   defaultConfig {
     versionName VERSION_NAME
+    vectorDrawables.useSupportLibrary = true
```

Comment:
There's no minSdk defined. Is it actually min 21 or are you lowering the functionality to work on 21? If it's min 21 then you shouldn't actually need this.

Reply:
Lowering to work with 21. If I specify an actual min in the build file it'll bump the min of projects that pull this in, right?

Reply:
Yes. You should at least specify 14 (which is what the support libs use). Without one you get 1 which adds some weird permission for compatibility reasons pre-3.